frontend/src/components/ui/streakFlameBadge.css

```
@@ -0,0 +1,98 @@
+  display: block;
+  transform-origin: 50% 70%;
+  animation: sfbFlicker 1.8s ease-in-out infinite;
+}
+
+/* Compositor-only animation props keep the loop off the main thread. */
+.sfb-glow,
+.sfb-flame {
+  will-change: transform, opacity;
+}
+
+@keyframes sfbGlowPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "sfbGlowPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)